tests/fuzz/sequence_compression_api.c

```diff
@@ -232,7 +232,7 @@ static size_t roundTripTest(void* result, size_t resultCapacity,
                             const void* src, size_t srcSize,
```

Comment:
We should test `ZSTD_compressSequencesAndLiterals()` here as well. We could just test both `ZSTD_compressSequences()` and `ZSTD_compressSequencesAndLiterals()`

Reply:
Fuzzer test for `ZSTD_compressSequencesAndLiterals()` added,
it piggy - backs on `ZSTD_compressSequences()`,
and gets triggered automatically whenever a compatible set of parameters is detected.